README.md

```diff
@@ -56,6 +56,7 @@ config = {
   cors: {
     allowedOrigins: ['localhost', 'lvh.me'],
     credentials: true, // Adds cors needed for exchanging cookies over https.
+    starPrefixes: ['/publicEndpoints/foo/bar'] // ctx.path prefixes that will return access-control-allow-origin: *
```

Comment:
Maybe allowed Prefixes?
```suggestion
    allowedPrefixes: ['/publicEndpoints/foo/bar'] // ctx.path prefixes that will return access-control-allow-origin: *
```

Reply:
Not sure. Because what is `allowedPrefixes`? it is a bit misleading because we also have `allowedOrigins`.

Maybe `publicPrefixes`? WDYT?

Reply:
I like it better than starPrefixes

Reply:
+1